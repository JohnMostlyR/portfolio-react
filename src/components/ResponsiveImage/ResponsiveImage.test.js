import React from 'react';
import renderer from 'react-test-renderer'

import ResponsiveImage from './index';

describe('ResponsiveImage', () => {
  it('Should render and match the snapshot', () => {
    const tree = renderer.create(<ResponsiveImage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});