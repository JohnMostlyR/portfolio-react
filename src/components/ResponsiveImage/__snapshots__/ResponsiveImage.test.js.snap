// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResponsiveImage Should render and match the snapshot 1`] = `
<div
  className="sc-bwzfXH uTxCW sc-bdVaJa foCoAy"
>
  <img
    alt={undefined}
    src={undefined}
  />
</div>
`;
